addons/mrp/models/mrp_production.py

```diff
@@ -781,6 +794,19 @@ def action_toggle_is_locked(self):
         self.is_locked = not self.is_locked
         return True
 
+    def action_product_forecast_report(self):
+        self.ensure_one()
+        action = self.product_id.action_product_forecast_report()
+        action['context'] = {
+            'active_id': self.product_id.id,
+            'active_model': 'product.product',
+            'move_to_match_id': self.move_finished_ids[0].id,
```

Comment:
You should filter on the product to produce (Check  `_compute_move_byproduct_ids`)